packages/sdk/python/tests/test_setup.py

```
@@ -0,0 +1,215 @@
+    name = "test-provider"
+
+    def __init__(self) -> None:
+        self.status_calls = 0
+
+    def proxy(self, request: dict[str, Any]) -> dict[str, Any]:
+        return {"status": 200, "headers": {}, "data": {}}
+
+    def health_check(self, connection_id: str) -> bool:
+        return True
+
+    def create_connect_session(self, input: dict[str, Any]) -> dict[str, Any]:
```

Ruff (0.15.18)

[error] Function argument `input` is shadowing a Python builtin

(A002)